docs/includes/usage-examples/RunCommandTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\DB;
+use MongoDB\Driver\Cursor;
+use MongoDB\Laravel\Tests\TestCase;
+
+class RunCommandTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testRunCommand(): void
+    {
+        // phpcs:disable
+        // begin-command
+        $cursor = DB::connection('mongodb')
+            ->command(['listCollections' => 1]);
+
+        foreach ($cursor as $coll) {
+            echo $coll['name'] . '<br>';
+        }
+
+        // end-command
+        // phpcs:enable
```

Comment:
Question for @GromNaN:
I'm not able to get the tests to pass since printing any output causes a CI test failure. Is there a way to keep the output code and pass the CI tests?

Reply:
I fixed the test by adding `$this->expectOutputRegex('/<br>/');`
The `phpcs:disable` statement was not necessary.